src/js/program/media-controller.js

```diff
@@ -303,7 +303,10 @@ export default class MediaController extends Eventable {
     }
 
     set position(pos) {
-        this.provider.seek(pos);
+        const { provider } = this;
+        const { fastSeek } = provider;
+        const seek = this.model.get('scrubbing') && fastSeek ? fastSeek : provider.seek;
+        seek(pos);
```

Comment:
We have to call `provider.seek` for `this` to refer to the provider. With these change it will refer to `window`.

Let's make it simpler:

```js
    const { provider } = this;
    if (this.model.get('scrubbing') && provider.fastSeek) {
        provider.fastSeek(pos);
    } else {
        provider.seek(pos);
    }
```